com.jetbrains.IntelliJ-IDEA-Ultimate.json

```diff
@@ -0,0 +1,72 @@
+{
+  "app-id": "com.jetbrains.IntelliJ-IDEA-Ultimate",
+  "command": "idea.sh",
+  "runtime": "org.freedesktop.Platform",
```

Comment:
@x80486 You should also use the SDK rather than the Platform here, as https://github.com/flathub/com.visualstudio.code.oss/blob/c73a7a723a397f39eb6c562de6cd6b5acffe40fb/com.visualstudio.code.oss.json#L14280 🤦

Reply:
Damn right! That was the issue! Thanks!